src/group_theory/subgroup/basic.lean

```diff
@@ -1284,6 +1284,23 @@ lemma le_normalizer_map (f : G →* N) :
     simp [hy, hyH, mul_assoc] }
 end
 
+variable (G)
+
+/-- Every proper subgroup `H` of `G` is a proper normal subgroup of the normalizer of `H` in `G`. -/
+def normalizer_condition := ∀ (H : subgroup G), H < ⊤ → H < normalizer H
```

Comment:
This puts it into the `subgroup` namespace. Reasonable?

Reply:
I'm inclined to put it in the root namespace or the `group` namespace. Because it's a property of `G`, not of subgroups of `G`.

Reply:
I agree. Should I just slap `_root_.` onto it, or move it elsewhere in the file, or close and open the namespace? The vincinity to `normalizer` is probably good.

Reply:
Just add `_root_.`. I agree that it's in a pretty good spot.